arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -301,6 +301,15 @@ void DatabaseFeature::collectOptions(
       .setIntroducedIn(30807)
       .setIntroducedIn(30902);
 
+  options
+      ->addOption("--database.max-databases",
+                  "The maximum number of databases that can exist in parallel.",
+                  new options::SizeTParameter(&_maxDatabases))
+      .setLongDescription(R"(If the maximum number of databases is reached, no
+additional databases can be created in the deployment. In order to create additional
+databases, other databases need to be removed first.")")
+      .setIntroducedIn(31200);
```

Comment:
Shouldn't this be: 
```suggestion
      .setIntroducedIn(31120);
```

Reply:
Or indeed `setIntroducedIn(31102);`?

Reply:
I think @neunhoef is correct, this should be: `.setIntroducedIn(31102);`